skins/laika/src/components/pages/donation_form/Address.vue

```diff
@@ -1,10 +1,10 @@
 <template>
 	<div id="addressForm" class="column is-full">
 		<payment-bank-data v-if="isDirectDebit" :validateBankDataUrl="validateBankDataUrl" :validateLegacyBankDataUrl="validateLegacyBankDataUrl"></payment-bank-data>
-		<address-type v-on:address-type="setAddressType( $event )"></address-type>
+		<address-type v-on:addressType="setAddressType( $event )"></address-type>
```

Comment:
Shouldn't we make our events strings snake-case to have snake-cased event handler attributes? I've done this with other events and did not pay attention to the address-type event.